modules/alloy/src/main/ts/ephox/alloy/api/Main.ts

```diff
@@ -18,9 +18,11 @@ import * as Bubble from '../positioning/layout/Bubble';
 
 import * as Layout from '../positioning/layout/Layout';
 import * as LayoutInside from '../positioning/layout/LayoutInside';
+import { contextBarTransitionClass } from '../positioning/layout/LayoutLabels';
```

Comment:
I really don't think this should be getting shipped as part of alloy's external API.

Reply:
What method would you prefer to use? Just hardcoding it in the theme?